.mailmap

```diff
@@ -1021,6 +1021,7 @@ Matthias Toews <mat.toews@googlemail.com>
 Mauro Garavello <mauro.garavello@unimib.it>
 Max Hutchinson <maxhutch@gmail.com>
 Maxence Mayrand <35958639+maxencemayrand@users.noreply.github.com>
+Maximilian F. Hasler <MFHasler+sympy@memiad.eu> m-f-h <github@dsi972.fr>
```

Comment:
You should add this as part of your other pull request rather than as a pull request on its own.